### DISCUSSION TITLE
Streamer Camera

### DISCUSSION THREAD

**Discussion by IhaveUsername** [nitrog0d/DesktopCamera#80] [General]
Is there a streamer mode for this camera? I am filming shows in VRchat and in VR mode there is streamer camera which would stream whatever camera sees to the VRchat window, allowing me to record footage with no UI, nametags, soft movement, etc. Can this be enables for non-vr players in this mod?

**Comment by YukiAyase** [comment]
Adding that is a bit of a problem for desktop since turning on streamer camera **will** make your VRChat window as the camera's screen itself and you will not be able to see any of the UI elements at all in this mode
Afaik there is no way of making a streamer camera possible for desktop users
